tests/coalaCITest.py

```diff
@@ -137,3 +137,41 @@ def test_fail_acquire_settings_debug(self):
                 r'were not provided.'
         ):
             self.test_fail_acquire_settings(debug=True)
+
+    def test_limit_files(self):
+        sample_text = '\t#include <a>'
+        match = open('match.cpp', 'w')
```

Comment:
use `with open(...):` syntax

Reply:
actually I think we also have a temporary file context manager, which is used by other tests.

Reply:
temporary file context manager doesn't support named files, AFAIK, it creates a temp file only if `name=None`